maintainers/maintainer-list.nix

```diff
@@ -4297,6 +4297,11 @@
     githubId = 10998835;
     name = "Doron Behar";
   };
+  dotemup = {
+    name = "Dote";
+    github = "dotemup";
+    githubId = 11077277;
```

Comment:
It would be nice to have another mean of contact

Reply:
Updated in latest.